test/modules/aws-codebuild-integration.ts

```diff
@@ -306,31 +306,16 @@ phases:
 
   it('apply codebuild_project creation', commit());
 
-  it('starts a transaction', begin());
-
   it(
     'start and wait for build',
     query(
       `
-    INSERT INTO codebuild_build_import (project_name)
-    VALUES ('${dbAlias}');
-  `,
-      undefined,
-      true,
-      () => ({ username, password }),
-    ),
-  );
-
-  it('apply build start', commit());
-
-  it(
-    'check build imports is empty',
-    query(
-      `
-    SELECT * FROM codebuild_build_import
-    WHERE project_name = '${dbAlias}';
+      SELECT * FROM start_build('${dbAlias}', '${region}');
```

Comment:
Don't we need an `iasql_begin` here? Wondering what will happen in case cron tick happens

Reply:
that's an rpc, so was not enclosed in begin/commit, follows the same logic as others